selfdrive/test/test_onroad.py

```diff
@@ -37,7 +37,7 @@
   "./_sensord": 7.0,
   "selfdrive.controls.radard": 4.5,
   "selfdrive.modeld.modeld": 8.0,
-  "./_dmonitoringmodeld": 5.0,
+  "selfdrive.modeld.dmonitoringmodeld": 8.0,
```

Comment:
This PR increases cpu usage for dmonitoringmodeld from 5% to 8%, hopefully this is within acceptable limits.